types.ts

```diff
@@ -42,20 +45,20 @@ export interface Broadcaster {
 };
 
 export interface PriceDataFetched {
-  symbol: string;
+  symbol: string; //symbol or token?
```

Comment:
I would still use "symbol" here because we already store not only tokens (e.g. stocks)